homework/yaryna/bubble_sorting_hw_8.py

```diff
@@ -0,0 +1,36 @@
+sequence = [{"value": 67},
+{"value": -18},
+{"value": 89},
+{"value": 0},
+{"value": 6},
+{"value": 12},
+{"value": -50},
+{"value": 100},
+{"value" : -100}]
+
+def bubble_sort(seq,key, reverse):
+    '''sort any sequence
+
+    Keyword arguments:
+    seq --  unsorted sequence
+    key -- selector of sequence item used for sorting 
+    reverse -- if True (default) sorted from min to max
+               if False sorted from max to min
+
+    Returns: sorted sequence 
+    '''
+    seq_lst = list(sequence)
+    skey = key if key is not None else lambda elem: elem
```

Comment:
may be better to have `lambda elem: elem` as default function argument to avoid extra logic